ci/code_checks.sh

```diff
@@ -121,86 +121,86 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     # Check for imports from pandas.core.common instead of `import pandas.core.common as com`
     # Check for imports from collections.abc instead of `from collections import abc`
     MSG='Check for non-standard imports' ; echo $MSG
-    invgrep -R --include="*.py*" -E "from pandas.core.common import" pandas
-    invgrep -R --include="*.py*" -E "from pandas.core import common" pandas
-    invgrep -R --include="*.py*" -E "from collections.abc import" pandas
-    invgrep -R --include="*.py*" -E "from numpy import nan" pandas
+    invgrep -R --include='*.py*' -E "from pandas.core.common import" pandas
```

Comment:
Why this change from double to single backticks? 
We shouldn't be changing things just for the sake of changing lines ..

Reply:
> We shouldn't be changing things just for the sake of changing lines ..

Ok, I understand